packages/astro/src/core/build/index.ts

```diff
@@ -35,32 +38,30 @@ class AstroBuilder {
 	private origin: string;
 	private routeCache: RouteCache;
 	private manifest: ManifestData;
-	private viteServer?: vite.ViteDevServer;
-	private viteConfig?: ViteConfigWithSSR;
+	private timer: Record<string, number>;
 
 	constructor(config: AstroConfig, options: BuildOptions) {
-		applyPolyfill();
-
```

Comment:
Was this intentional?

Reply:
Yea, kind of unrelated but this moves `applyPolyfill()` into the CLI function that gets called (`add()`, `build()`, etc) to match all of the other commands. 

Also it's just a bit odd to have a side-effect like this happen in the constructor of another object., imo

Reply:
That's fair!